common/include/pcl/common/fft/kiss_fft.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <math.h>
-#include <string.h>
+#include <stdlib.h> // NOLINT
+#include <stdio.h>  // NOLINT
+#include <math.h>   // NOLINT
+#include <string.h> // NOLINT
```

Comment:
kiss_fft is C header. It's probably best to apply no lint to places which include kiss_fft.h

Reply:
Unless `// NOLINT` is on these lines, `clang-tidy` tries to change these to C++ headers (`<cstdlib>`, etc)

Reply:
I think there's a header-filter regex